### PR TITLE
[NO-TICKET] Sass to CSS migration guide

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

- Added a _Sass to CSS_ migration guide page
- Added a reference to it in the Release 6.0 blog post
- Narrowed the gap between page headers and page content when the GitHub/Storybook/Sketch links are not present

## How to test

1. `yarn start` and navigate to `Migration Guides > Sass to CSS`

### CODE REVIEW

File: packages/docs/content/blog/release-6.0.mdx [hunk 1]
```diff
@@ -19,17 +19,17 @@ Previously we had two options for teams consuming our design system styles: CSS
...
 
 <div className="ds-u-measure--wide ds-u-text-align--center">
   <img src={ssnInput} alt="Animated screen capture of inputting a Social Security Number using the new label-mask pattern" className="ds-u-display--inline-block" />
 </div>
 
-Hiding the SSN is previously something application teams had to implement themselves to use the SSN field and maintain their users' privacy. You can [read more about the label-mask pattern](https://design.cms.gov/components/text-field/label-masked-field/)—what it is and why we built it—on the new documentation page.
+Hiding the SSN is previously something application teams had to implement themselves to use the SSN field and maintain their users' privacy. You can [read more about the label-mask pattern](/components/text-field/label-masked-field/)—what it is and why we built it—on the new documentation page.
```
Comment (zarahzachz) [inline packages/docs/content/blog/release-6.0.mdx:32]: "—what" needs a space I believe

Comment (zarahzachz) [reply packages/docs/content/blog/release-6.0.mdx:32]: Same with "it—on"

File: packages/docs/content/blog/release-6.0.mdx [hunk 2]
```diff
@@ -54,15 +54,15 @@ In our [3.6.0 release](https://github.com/CMSgov/design-system/releases/tag/%40c
   <img src={swatches} alt="Color ramps from the 'System Color Tokens' page, showing a range of light to dark shades of a color" className="ds-u-display--inline-block" width="400" />
 </div>
 
-In this release we've added a page to the design system to show all color tokens in an easy-to-read and scannable way. We're exposing this base set of tokens to better highlight the design decisions made within a particular theme. Think of them as [a complete box of crayons](https://design.cms.gov/foundation/system-color-tokens/?theme=core). Our themes make use of certain colors from the crayon box but not all of them.
+In this release we've added a page to the design system to show all color tokens in an easy-to-read and scannable way. We're exposing this base set of tokens to better highlight the design decisions made within a particular theme. Think of them as [a complete box of crayons](/foundation/system-color-tokens/?theme=core). Our themes make use of certain colors from the crayon box but not all of them.
```
Comment (zarahzachz) [inline packages/docs/content/blog/release-6.0.mdx:57]: is "?theme=core" going to redirect ppl to core even if they're reading this post on Medicare or something?

File: packages/docs/content/blog/release-6.0.mdx [hunk 3]
```diff
@@ -54,15 +54,15 @@ In our [3.6.0 release](https://github.com/CMSgov/design-system/releases/tag/%40c
...
 
 ### Streamlining the ZIP Code design pattern
 
 We dove deep into a dozen CMS, HealthCare, and Medicare products, and what we discovered was truly eye-opening. We found a wide range of design patterns used for ZIP code input fields, but we also found some commonalities that can lead to a more efficient and user-friendly experience.
 
 Some products used a 5-digit ZIP code pattern, others used a 9-digit ZIP code pattern for stricter data validation. We also discovered services using an autocomplete feature for users to confirm their geo-location.
 
-ZIP codes are an important part of how we search for services and determine eligibility at CMS. Overall, we're thrilled to bring these design insights to the table and excited to see the impact they'll have on our user's experiences. [ZIP code pattern guidance](https://design.cms.gov/patterns/zip-codes/?theme=core).
+ZIP codes are an important part of how we search for services and determine eligibility at CMS. Overall, we're thrilled to bring these design insights to the table and excited to see the impact they'll have on our user's experiences. [ZIP code pattern guidance](/patterns/zip-codes/?theme=core).
```
Comment (zarahzachz) [inline packages/docs/content/blog/release-6.0.mdx:65]: I believe our guidance is links should have action language to tell ppl what to expect, but "[ZIP code pattern guidance]" doesn't adhere to that

File: packages/docs/content/foundation/typography/content.mdx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+title: Content
+order: 0
+intro: The "ds-content" class provides an easy way to style written content on your page
+core:
+  githubLink: design-system/src/styles/_base.scss
+---
+
+For pages that are mainly about written content or places where you want to use a standard mapping of heading-level styles to semantic heading levels , we have a special `ds-content` CSS class. Applying this class to an element that surrounds your content will automatically apply design system heading styles to the appropriate `h1`, `h2`, `h3`, `h4`, `h5`, and `h6` tags as well as `ol` and `ul` list tags.
```
Comment (zarahzachz) [inline packages/docs/content/foundation/typography/content.mdx:9]: Unnecessary space here "heading levels , we"

File: packages/docs/content/foundation/typography/headings.mdx [hunk 5]
```diff
@@ -164,12 +164,7 @@ This is an optional header design.
 
 ### The `ds-content` class
 
-For pages that are mainly about written content or places where you want to use a standard mapping of heading-level styles to semantic heading levels , we have a special `ds-content` CSS class. Applying this class to an element that surrounds your content will automatically apply design system heading styles to the appropriate `h1`, `h2`, `h3`, `h4`, `h5`, and `h6` tags.
-
-<StorybookExample
-  componentName="content-class"
-  storyId="foundations-typography-headings--content-class"
-/>
+For pages that are mainly about written content or places where you want to use a standard mapping of heading-level styles to semantic heading levels , consider using the [`ds-content`](/foundation/typography/content/) class.
```
Comment (zarahzachz) [inline packages/docs/content/foundation/typography/headings.mdx:167]: Unnecessary space "g levels , consider"

File: packages/docs/content/blog/release-6.0.mdx (hunk 1 above)
Comment (pwolfert) [reply packages/docs/content/blog/release-6.0.mdx:32]: It's hard to see in the monospace font, but that character is a dash, so there's actually not supposed to be any spaces around it

File: packages/docs/content/blog/release-6.0.mdx (hunk 3 above)
Comment (pwolfert) [reply packages/docs/content/blog/release-6.0.mdx:65]: I'll change it and fix the URL. How about
> See ZIP code pattern guidance?

**Review by brentschneider — APPROVED** [review]
Reads very clear to me, excellent write up

**Pull request merged**